src/SFA.DAS.Commitments.Api.IntegrationTests/ApiHost/IntegrationTestServer.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Reflection;
+using System.Web;
+using System.Web.Http;
+using System.Web.Http.Dispatcher;
+using System.Web.SessionState;
+using Microsoft.Owin.Testing;
+using Owin;
+using StructureMap;
+using SFA.DAS.Commitments.Api.Controllers;
+using SFA.DAS.Commitments.Api.DependencyResolution;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.Commitments.Api.IntegrationTests.ApiHost
+{
+    class IntegrationTestServer
```

Comment:
Could you stick a class modifier on this?

Reply:
Added the public modifier.